mobile-app/app/api/wallet/service_provider.ts

```diff
@@ -1,14 +1,31 @@
 import { SecuredStoreAPI } from "@api";
+import { CustomServiceProviderType } from "@contexts/CustomServiceProvider";
 
 const KEY = "WALLET.SERVICE_PROVIDER_URL";
 
-async function set(url: NonNullable<string>): Promise<void> {
-  await SecuredStoreAPI.setItem(KEY, url);
+async function set(
+  url: NonNullable<string>,
+  type: CustomServiceProviderType = CustomServiceProviderType.DVM,
+): Promise<void> {
+  await SecuredStoreAPI.setItem(`${KEY}.${type}`, url);
 }
 
-async function get(): Promise<string | undefined> {
-  const val = await SecuredStoreAPI.getItem(KEY);
-  return val != null ? val : undefined;
+async function get(
+  type: CustomServiceProviderType = CustomServiceProviderType.DVM,
+): Promise<string | undefined> {
+  const PROVIDER_KEY = `${KEY}.${type}`;
+  const val = await SecuredStoreAPI.getItem(PROVIDER_KEY);
+
```

Comment:
if we find value here then lets just return it, no need to check one more condition, What you think @lykalabrada ?

Reply:
Agree, that's better 🚀